### ISSUE TITLE
Can not find and add files in  in rviz panal !!!

### ISSUE THREAD

**Issue by mechaliomar** [ethz-asl/mav_voxblox_planning#39]
@helenol Thanks for your respond. I solved that problem by lunching roscore in one terminal and rviz in another terminal. But still have a problem, I cant do the following:

In rviz, select Panels -> Add New Panel and select Planning Panel:
I just have the following choices
-rviz
-rviz_plugin_tutorials

I can't find other files as you mentioned in your post

![IMG_20190417_204538](https://user-images.githubusercontent.com/15894806/56399514-f4912680-6280-11e9-93e2-d92f17f0a0c7.jpg)
![IMG_20190417_204514](https://user-images.githubusercontent.com/15894806/56399515-f4912680-6280-11e9-89d4-36e94b76c5bb.jpg)

**Comment by gouyiyong** [comment]
you should source your workspace, and then setup the rviz.

**Comment by zhengyu8023** [comment]
Hello ! Have you solved this problem? I met the same problem.

**Comment by mechaliomar** [comment]
@zhengyu8023  not yet, I still couldn't solve it

**Comment by logindiamant** [comment]
In terminal, where you want to run rviz, go to devel folder of mav_voxblox_planning, and run source setup.bash
Then run rviz

**Comment by zhkmxx9302013** [comment]
the panel is not at here, you should maximize the rviz window, then find the panels on the title bar.
![image](https://user-images.githubusercontent.com/10081554/60167274-5b106300-9835-11e9-82e9-172b9b6c74ee.png)

**Comment by wntun** [comment]
Yes. have to source first '$ source ~/catkin_ws/devel/setup.bash' before type '$ rviz' in the same terminal.
